osu.Game.Tests/Visual/Background/TestSceneUserDimContainer.cs

```diff
@@ -313,7 +332,7 @@ private void load(OsuConfigManager config)
                 config.BindWith(OsuSetting.BlurLevel, BlurLevel);
             }
 
-            public bool IsBackgroundDimmed() => ((FadeAccessibleBackground)Background).CurrentColour == OsuColour.Gray(1 - (float)DimLevel.Value);
+            public bool IsBackgroundDimmed() => ((FadeAccessibleBackground)Background).CurrentColour == OsuColour.Gray(Math.Min(1 - (float)((FadeAccessibleBackground)Background).DimLevel, 1));
```

Comment:
I'm a bit worried about the argument to `OsuColour.Gray` potentially going below 0 - might be worth throwing in an assert or a clamp?